### PR TITLE
Design Rework #2 - Colors & Layouts

### PR DESCRIPTION
**Pull request by LuisMossburger**
**Description**
See also #100 and #118 for progress.
This merges reworked, now clean layouts and colors matching our palettes and suggestions from Alexander Bazo. Please see the single commits for details, as more things were done here. In short:
* Clean design & colors for accent
* Consistent use of spacing, colors and elements (search bar, buttons)
* Sizing of list items, header and buttons reworked

Next up: Toasts & Help, Home Fragment, "design sugar".

**Affects**
Complete app design.

**Notes for Reviewer**
Please review & comment/approve - maybe don't be too nitpicky as I think the changes will need to be discussed in a future team meeting instead of between the two of us here :)

### CODE REVIEW

File: BibBuddy/app/src/main/res/layout/add_button.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            android:tooltipText="@string/add"
+            android:contentDescription="@string/add"
+            android:src="@drawable/add"
+            app:tint="@color/flirt"
+            app:backgroundTint="@color/white"
+            app:useCompatPadding="true"
+            tools:targetApi="o"
+            tools:ignore="MissingConstraints" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (claudia3) [inline BibBuddy/app/src/main/res/layout/add_button.xml:24]: Please add a newline at the end of the file.
```suggestion
</androidx.constraintlayout.widget.ConstraintLayout>

```

File: BibBuddy/app/src/main/res/layout/confirm_button.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            android:tooltipText="@string/confirm"
+            android:contentDescription="@string/confirm"
+            android:src="@drawable/ic_check"
+            app:tint="@color/tiffany"
+            app:backgroundTint="@color/white"
+            app:useCompatPadding="true"
+            tools:targetApi="o"
+            tools:ignore="MissingConstraints" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (claudia3) [inline BibBuddy/app/src/main/res/layout/confirm_button.xml:24]: Please add a newline at the end of the file.
```suggestion
</androidx.constraintlayout.widget.ConstraintLayout>

```

**Comment by claudia3** [comment]
In our last discussion with Alexander Bazo he told us that we should not use a search button next to the searchInput field. So please remove it. He told us that the Icon in the keyboard for the search is sufficient.

**Comment by LuisMossburger** [comment]
> 
> 
> In our last discussion with Alexander Bazo he told us that we should not use a search button next to the searchInput field. So please remove it. He told us that the Icon in the keyboard for the search is sufficient.

That's true, however we decided to discuss this in the next team meeting on this Tuesday - so I would leave it until then if you don't mind.

File: BibBuddy/app/src/main/res/layout/add_button.xml (hunk 1 above)
Comment (LuisMossburger) [reply BibBuddy/app/src/main/res/layout/add_button.xml]: Done.

File: BibBuddy/app/src/main/res/layout/confirm_button.xml (hunk 2 above)
Comment (LuisMossburger) [reply BibBuddy/app/src/main/res/layout/confirm_button.xml]: Done.

**Comment by claudia3** [comment]
> > In our last discussion with Alexander Bazo he told us that we should not use a search button next to the searchInput field. So please remove it. He told us that the Icon in the keyboard for the search is sufficient.
> 
> That's true, however we decided to discuss this in the next team meeting on this Tuesday - so I would leave it until then if you don't mind.

Ok

**Review by claudia3 — APPROVED** [review]
The new consistent design looks good.

**Pull request merged**